### DISCUSSION TITLE
Oxyplot new release date

### DISCUSSION THREAD

**Discussion by sathish-mtts** [oxyplot/oxyplot#2071] [Q&A] [answered]
Please let us know when the new release will be happening for this year 2024.

**Comment by VisualMelon** [comment, marked as answer]
Can't put a date on it, but probably 2.2.0 will go out once we have a solution for #2061

**Reply by sathish-mtts** [reply]
Thank you very much for your update...

**Reply by n-tiz** [reply]
Hello @VisualMelon,

Any news about the solution for #2061 ?
It's been a while since there was a release of OxyPlot, which is a bit unfortunate.

Thank you for your continued hard work on Oxyplot 😊

**Reply by VisualMelon** [reply]
No news; not got oodles of time at the moment. Feel free to keep pinging me to remind me what's most important to everyone.

**Comment by Dimigergo** [comment]
Hello @VisualMelon! 
When do we get the 2.1.3 release? Some bug fixes and features for me are in this version.

Thanks for your answer!

**Reply by sathish-mtts** [reply]
Hello,

Last update is nearly 2 years. Since some of our scanning tools shows our product has operational risk.

Could you please consider this in mind a release the update ASAP.

Thanks,
Sathish P

**Reply by n-tiz** [reply]
Hello,

I agree, I think it's time, and it would be a good idea to do a release !

**Reply by VisualMelon** [reply]
I'll try to put something up this weekend

**Reply by sathish-mtts** [reply]
> I'll try to put something up this weekend

We are eagerly waiting for this release from long time...Thank you very much for your support

**Reply by VisualMelon** [reply]
Pushed a candidate 2.1.3 to my test feed https://www.myget.org/F/melonoxyplottesting which includes #2099

Not decided on naming or what exactly will be in the proper pushed version (mostly did this to test my scripts still work)

**Reply by VisualMelon** [reply]
@Jonarw anything that should be tweaked with Skia packages before a release? (or any other input)

**Reply by VisualMelon** [reply]
Linking #2101

**Reply by Jonarw** [reply]
I think we are fine with Skia at the moment, as we are on the latest stable 2.88.8. So from my side, there is nothing that would prevent us from releasing 2.2 👍

**Reply by VisualMelon** [reply]
2.2.0 has been released to Nuget. Please ping me if anything isn't working. I'll try to work out how the release notes work later in the week.

**Reply by sathish-mtts** [reply]
Thank you very much for your quick response and support for considering my point and released the new version...
